### ISSUE TITLE
Willing to perform motion forecasting

### ISSUE THREAD

**Issue by philipperbd** [wholebody3d/wholebody3d#13]
Hi, really nice work. It was a pleasure to see you present your work. I would like to work on motion forecasting. Have you tried any model for this task ? And what .json should I use to ? Thanks

**Comment by Zhuyue0324** [comment]
For motion forecasting task, our dataset is not a good choice because we don't organize data by video sequences, meaning there are only a few successive frame sequences.

But if you still want to try something from it, I would suggest you can try using image paths (from rgb to 3d) to find the continuous frames, and then using the correspondent image/2d/3d data depending on your task.

**Comment by philipperbd** [comment]
I thought you extracted whole bodies from all frames (=all videos) from Human3.6m. Am i wrong ?

But if it is just about finding the corresponding frames for each video I could handle it.

**Comment by Zhuyue0324** [comment]
No, i didn't extract all frames. Many frames are not used because they don't have every keypoint be detected at least once in any view by 2D detector
